rust/kcl-lib/src/std/chamfer.rs

```diff
@@ -47,6 +74,53 @@ async fn inner_chamfer(
         )));
     }
 
+    if angle.is_some() && second_length.is_some() {
+        return Err(KclError::new_semantic(KclErrorDetails::new(
+            "Cannot specify both an angle and a second length. Specify only one.".to_string(),
+            vec![args.source_range],
+        )));
+    }
+
+    let second_distance = second_length.clone().map(|x| LengthUnit(x.to_mm()));
```

Comment:
You can avoid the clone here by moving `let strategy` up above here. Another option is to do:

```suggestion
    let second_distance = second_length.as_ref().map(|x| LengthUnit(x.to_mm()));
```